src/crafting_gui.cpp

```diff
@@ -936,6 +941,7 @@ const recipe *select_crafting_recipe( int &batch_size_out )
                 { 't', _( "soldering iron" ), _( "<color_cyan>tool</color> required to craft" ) },
                 { 'm', _( "yes" ), _( "recipes which are <color_cyan>memorized</color> or not" ) },
                 { 'P', _( "Blacksmithing" ), _( "<color_cyan>proficiency</color> used to craft" ) },
+                { 'l', _( "5" ), _( "<color_cyan>difficulty</color> of the recipe as a number or range" ) },
```

Comment:
`_( "5" )` -> `"5"`
I started translating and noticed that it was proposed to translate "5". I think this is unnecessarily :)

Reply:
Some languages might use different symbols for numbers, though I don't really know if that would work well when using the filter, so I left it as a translatable string just in case someone needs to change the example.